systers_portal/templates/blog/snippets/tags_sidebar.html

```diff
@@ -0,0 +1,12 @@
+{% load guardian_tags %}
+{% if user.is_authenticated and user.is_active %}
+  {% get_obj_perms user for community as "community_perms" %}
```

Comment:
Since we need to check against a Django permissions lines 3-4 will change. See documentation for Django Permission class and it's usage in templates.